addons/sale/report/sale_report_views.xml

```diff
@@ -101,6 +101,36 @@
         <field name="help">This report performs analysis on your quotations and sales orders. Analysis check your sales revenues and sort it by different group criteria (salesman, partner, product, etc.) Use this report to perform analysis on sales not having invoiced yet. If you want to analyse your turnover, you should use the Invoice Analysis report in the Accounting application.</field>
     </record>
 
+    <record id="action_order_report_salesperson" model="ir.actions.act_window">
+        <field name="name">Sales Analysis By Salespersons</field>
+        <field name="res_model">sale.report</field>
+        <field name="view_mode">graph,pivot</field>
+        <field name="view_id"></field>  <!-- force empty -->
+        <field name="search_view_id" ref="view_order_product_search"/>
+        <field name="context">{'search_default_User': 1}</field>
+        <field name="help">This report performs analysis on your quotations and sales orders. Analysis check your sales revenues and sort it by different group criteria (salesman, partner, product, etc.) Use this report to perform analysis on sales not having invoiced yet. If you want to analyse your turnover, you should use the Invoice Analysis report in the Accounting application.</field>
+    </record>
+
+    <record id="action_order_report_customers" model="ir.actions.act_window">
+        <field name="name">Sales Analysis By Customers</field>
+        <field name="res_model">sale.report</field>
+        <field name="view_mode">graph,pivot</field>
+        <field name="view_id"></field>  <!-- force empty -->
+        <field name="search_view_id" ref="view_order_product_search"/>
+        <field name="context">{'search_default_Customer': 1}</field>
+        <field name="help">This report performs analysis on your quotations and sales orders. Analysis check your sales revenues and sort it by different group criteria (salesman, partner, product, etc.) Use this report to perform analysis on sales not having invoiced yet. If you want to analyse your turnover, you should use the Invoice Analysis report in the Accounting application.</field>
+    </record>
+
+    <record id="action_order_report_products" model="ir.actions.act_window">
+        <field name="name">Sales Analysis Products</field>
+        <field name="res_model">sale.report</field>
+        <field name="view_mode">graph,pivot</field>
+        <field name="view_id"></field>  <!-- force empty -->
+        <field name="search_view_id" ref="view_order_product_search"/>
+        <field name="context">{'search_default_Sales':1, 'search_default_Product': 1, 'search_default_filter_order_date': 1}</field>
```

Comment:
Not 100% sure, but I'd say
```suggestion
        <field name="context">{'search_default_Product':1, 'group_by_no_leaf':1, 'group_by': 'Product, 'search_default_filter_order_date': 1}</field>
```

Reply:
Hello @vava-odoo, as per the specification given on task we need to redirect to the current report with product group by applied, if we will remove 'search_default_Sales' then sale order filter will no be apply for that menu.